include/seqan3/range/views/repeat.hpp

```diff
@@ -50,6 +50,20 @@ class repeat_view : public std::ranges::view_interface<repeat_view<value_t>>
     //!\brief The sentinel type is set to std::ranges::default_sentinel_t.
     using sentinel_type = std::ranges::default_sentinel_t;
 
+    /*!\name Associated types
+     *!\brief These associated types are needed in random_access_iterator.
```

Comment:
```suggestion
     *!\brief These associated types are needed in seqan3::detail::random_access_iterator.
```
That way there is a proper link in the documentation later on